src/services/ldap/index.js

```diff
@@ -108,18 +108,12 @@ module.exports = function () {
          * @return {Promise} resolves if successfully disconnected, otherwise
          * rejects with error
          */
-		_disconnect(config) {
-			return new Promise((resolve, reject) => {
-				if (!(config && config._id)) {
-					reject('Invalid config object');
-				}
-				this._getClient(config).unbind((err) => {
-					if (err) {
-						reject(err);
-					}
-					resolve();
-				});
-			});
+		disconnect(config) {
+			return this._getClient(config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected dangling '_' in '_getClient'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5433011)